exercises/binary-search/instructions.md

```diff
@@ -0,0 +1,28 @@
+# Instructions
+
+Your task is to implement a binary search algorithm.
+
+A binary search algorithm finds an item in a list by repeatedly splitting it in half, only keeping the half which contains the item we're looking for.
+It allows us to quickly narrow down the possible locations of our item until we find it, or until we've eliminated all possible locations.
+
+```exercism/caution
+Binary search only works when a list has been sorted.
+```
```

Comment:
The [admonition spec] says these should be 4 tildes as this doesn't seem like an exceptional circumstance: https://github.com/exercism/docs/blob/main/building/markdown/markdown.md#special-blocks-sometimes-called-admonitions